ledger-core/network/nds/msgdelivery/retries/staged.go

```diff
@@ -23,7 +23,7 @@ type StagedController struct {
 }
 
 type RetryStrategy interface {
-	Retry(ids []RetryID, repeatFn func(RetryID))
+	Retry(ids []RetryID, repeatFn func(RetryID), bulkOverflow func([]RetryID))
```

Comment:
bulkOverflow is not a clean name for me? What is overflow? "bulkRepeatFn" better.

Reply:
bulkOverflow is called for entries that were NOT retried, so it is very different than repeatFn 
entries, reported via bulkOverflow will have a bit shorter path to next retry

Reply:
Why interface should know something about self-usage? Overflow on the caller side should not affect the name in the interface. I agree that we need this option, my concern about naming

Reply:
because caller knows about inability of retry (==overflow), while provider knows how to handle it